src/test/java/overlap_maximization_use_case/TestCalculateSectionHoursInteractor.java

```diff
@@ -0,0 +1,55 @@
+//package overlap_crap_fix_locations_later.tests;
```

Comment:
Is there any reason why the tests here are commented out? If we can make them work before merging, that would be for the best.